packages/browser-repl/src/components/shell-output-line.tsx

```diff
@@ -81,6 +81,35 @@ export class ShellOutputLine extends Component<ShellOutputLineProps> {
       return <CursorIterationResultOutput value={value} />;
     }
 
+    if (type === 'ShowProfileResult') {
+      if (value.count === 0) {
```

Comment:
Looks like there is enough code for a new element type like these ones: https://github.com/mongodb-js/mongosh/tree/7655f4dfb95cfda1d5c18a9db5347725916e49fb/packages/browser-repl/src/components/types